mizar/obj/endpoint.py

```diff
@@ -462,6 +462,8 @@ def delete_network_policy_egress(self, cidr_type, cidr_table):
     def delete_network_policy_protocol_port_ingress(self, port_table):
         cidr_networkpolicy_list = []
         for item in port_table:
+            if item["protocol"] == "any" and item["port"] == "0":
```

Comment:
do this block mean the open-to-all policy never been deleted?

Reply:
You are right. I withdrawal the change here..